.eslintrc.js

```diff
@@ -23,11 +22,18 @@ module.exports = {
     '@typescript-eslint/class-name-casing': 'error',
     '@typescript-eslint/no-angle-bracket-type-assertion': 'error',
     '@typescript-eslint/no-inferrable-types': 'error',
-    '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
+    // @todo re-enable when ESLint has been updated
+    // '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
```

Comment:
I had to update `@typescript-eslint/parser`, but didn't want to mess with all the other ESLint dependencies in the same PR, as changed rules could have caused a lot of churn. Unfortunately it seems this rule freaks out now on unused variables in `types`, which uh, makes no sense.

I would try to update the ESLint stuff later in a separate effort, I expect that we can activate this rule then again!